tools/ProductConstructionService.ReproTool/Options/FullBackflowTestOptions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using CommandLine;
+using Microsoft.Extensions.DependencyInjection;
+using ProductConstructionService.ReproTool.Operations;
+
+namespace ProductConstructionService.ReproTool.Options;
+
+[Verb("backflow-test", HelpText = "Flows an existing VMR build to all repos in maestro-auth-test")]
+internal class FullBackflowTestOptions : Options
+{
+    [Option("build", HelpText = "Real VMR build from which we'll take assets from", Required = true)]
+    public required int BuildId { get; init; }
+
+    [Option("target-branch", HelpText = "Target branch in all repos", Required = true)]
+    public required string TargetBranch { get; init; }
```

Comment:
How does this work? Doesn't the branch differ per repo?
I think the branch you get from the dependency resolver should be the target branch.

Reply:
Should be optional I think. I added it because after doing the forward flow, I created a branch in every repo from the one I forward flew the code from. Maybe the forward flow command could do this, and output it at the end